LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -1193,9 +1193,16 @@ BriefSummary="Massively reduces weapon noise and the soldier's impact on mission
 TinySummary="Reduces mission infiltration time"
 LootTooltip="This weapon upgrade, when installed, will reduce the time the soldier requires to infiltrate during a mission. It also reduces weapon noise during combat."
 
+[CritUpgrade_Bsc X2WeaponUpgradeTemplate]
+BriefSummary="Increase critical hit chance by +5%".
+
+[CritUpgrade_Adv X2WeaponUpgradeTemplate]
+BriefSummary="Increase critical hit chance by +10%".
+
 [CritUpgrade_Sup X2WeaponUpgradeTemplate]
 FriendlyName="Elite Laser Sight"
 FriendlyNamePlural="Elite Laser Sights"
+BriefSummary="Increase critical hit chance by +15%."
```

Comment:
This line should also be independently wrapped with `; LWOTC Needs Translation` and also should use the ability tag.